### PR TITLE
Add JS Avo Logger example

### PR DESCRIPTION
**Pull request by alex-tpom6oh**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝Changed routes:


| [**/implementation/logs**](https://docs-git-js-avo-logger.teamavo.now.sh/implementation/logs) |
|:-:|
|<a href="https://docs-git-js-avo-logger.teamavo.now.sh/implementation/logs">Screenshot of /implementation/logs</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/ZFREXBtta54TtSYh6757DmeXLMg_.png">(view full size)</a> |







Commit 069d3acd9a20e227cb26c0d5170f4da5997d1975 (https://docs-git-js-avo-logger.teamavo.now.sh).

**Comment by logason** [comment]
Should we move this page from Tools to Reference > Avo Functions in the nav bar?

File: pages/implementation/logs.mdx [hunk 1]
```diff
@@ -38,6 +38,30 @@ Avo.initAvo(...)
 Avo.avoLogger = MyAvoLogger()
 ```
 
+### JavaScript
```
Comment (logason) [inline pages/implementation/logs.mdx:41]: Before we go into the code examples I think we should say: "Custom loggers are currently only supported in Swift and Javascript. Reach out to us at X if you want access to custom loggers in your programming languages

Comment (logason) [reply pages/implementation/logs.mdx:41]: Also: Add title like this above Swift as well

Comment (alex-tpom6oh) [reply pages/implementation/logs.mdx:41]: 👍

File: pages/implementation/logs.mdx [hunk 2]
```diff
@@ -9,60 +9,7 @@ import Link from '../../components/Link';
...
-    };
-
-    Avo.initAvo(
-      { env: env.DEV, ..., avoLogger: avoLogger },
-      ...
-    );
-```
-
-> Avo Logger is not yet supported on all the platforms. <Link passHref href="/help/troubleshooting"><a>Contact us</a></Link> if you want to use it on a particular platform that does not support it yet and we will enable the feature.
+Learn about controlling the logs output <Link passHref href="/implementation/custom-loggers"><a>in the reference senction</a></Link>.
```
Comment (logason) [inline pages/implementation/logs.mdx:12]:
```suggestion
Learn about controlling the logs output in Avo Functions <Link passHref href="/implementation/custom-loggers"><a>in the reference senction</a></Link>.
```

**Review by logason — APPROVED** [review]
LGTM 👍

**Pull request merged**